src/test/java/com/adyen/mirakl/cucumber/stepdefs/IdentityVerificationSteps.java

```diff
@@ -232,4 +232,14 @@ public void theDocumentsWillBeRemovedForEachOfTheUBOs() {
     public void eachUBOWillReceiveARemedialEmail(String title) throws Throwable {
         validationCheckOnReceivedEmails(title, shop);
     }
+
+
+    @And("^the shop is updated to tier1$")
+    public void updateShopToTier3() throws Throwable {
```

Comment:
```suggestion
    public void theShopIsUpdatedToTier1() throws Exception {
```

Is _throws Throwable_ necessary or can we use Exception?